pwiz_tools/Skyline/TestFunctional/PasteTransitionListTest.cs

```diff
@@ -315,5 +318,33 @@ private void TestPeptideOnlyDoc()
             RunUI(() => documentGrid.Close());
         }
 
+        private void TestMissingFragmentMz()
+        {
+            // Make sure we properly handle missing fragment mz info
+            var saveColumnOrder = Settings.Default.CustomMoleculeTransitionInsertColumnsList;
+            var text =  "Peptide Modified Sequence\tPrecursor m/z\tFragment Name\tProduct Charge\nPEPTIDER\t478.738\ty1\t1\nPEPTIDER\t478.738\ty2\t1";
+            var importDialog = ShowDialog<InsertTransitionListDlg>(SkylineWindow.ShowPasteTransitionListDlg);
+            var columnSelectDlg = ShowDialog<ImportTransitionListColumnSelectDlg>(() => importDialog.textBox1.Text =
+                text.Replace(".", CultureInfo.CurrentCulture.NumberFormat.NumberDecimalSeparator));
+
+            var errDlg = ShowDialog<MessageDlg>(columnSelectDlg.CheckForErrors);
+            RunUI(() =>
+            {
+                Assert.IsTrue(errDlg.Message.Contains(Resources.ImportTransitionListErrorDlg_ImportTransitionListErrorDlg_This_transition_list_cannot_be_imported_as_it_does_not_provide_values_for_) &&
+                              errDlg.Message.Contains(Resources.ImportTransitionListColumnSelectDlg_PopulateComboBoxes_Product_m_z),
+                    string.Format(

```

Comment:
You don't need to use "string.Format" for the message part of Asserts.
All of the Assert methods are designed so that they take "string message, params object[] parameters".
The idea is that "string.Format" is expensive and so you pass the format string and parameters so that "string.Format" only needs to be called if the test fails.
(It's a microscopic performance improvement, but that is why those methods all have that sort of signature, and also why methods like "Trace.TraceInformation" are like that).

Reply:
Good to know, I'll tidy that up so nobody folllows my bad example.